src/5-native-token-bridge/_INSTRUCTIONS.md

```diff
@@ -0,0 +1,248 @@
+
+Using the [`forge create`](https://book.getfoundry.sh/reference/forge/forge-create) command, we will deploy the [ERC20Destination.sol](./NativeTokenSource.sol) contract, passing in the following constructor arguments:
+
+```zsh
+forge create --rpc-url local-c --private-key $PK src/5-native-token-bridge/ERC20Destination.sol:ERC20Destination --constructor-args <TeleporterRegistry> <TeleporterManager> <SourceBlockchainID> <TokenSourceAddress> <TokenName> <TokenSymbol> <TokenDecimals>
+```
+
+- Teleporter Registry Address **(for C-Chain)**
+- Teleporter Manager (our funded address)
+- Source Blockchain ID (hexidecimal representation of our Subnet's Blockchain ID)
+- Token Source Address (address of NativeTokenSource.sol deployed on Subnet in the last step)
+- Token Name (input in the constructor of the [wrapped token contract](./ExampleWNATV.sol))
+- Token Symbol (input in the constructor of the [wrapped token contract](./ExampleWNATV.sol))
+- Token Decimals (uint8 integer representing number of decimal places for the ERC20 token being created. Most ERC20 tokens follow the Ethereum standard, which defines 18 decimal places.)
+
+To get the `Source Blockchain ID` in hexidecimal format, which in this example is the BlockchainID of your Subnet, run:
+
+```zsh
+avalanche subnet describe mysubnet
+```
+
+For example, this contract deployment could be entered into your terminal as:
+
+```zsh
+forge create --rpc-url local-c --private-key $PK src/5-native-token-bridge/ERC20Destination.sol:ERC20Destination --constructor-args 0xAd00Ce990172Cfed987B0cECd3eF58221471a0a3 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC 0xbcb8143686b1f0c765a1404bb94ad13134cafa5cf56f181a3a990ba21b1151b9 0x17aB05351fC94a1a67Bf3f56DdbB941aE6c63E25 "Wrapped NATV" "WNATV" 18
+```
+
+Note the address the source contract was "Deployed to".
+
+## Bridge the Token Cross-chain
+
+Now that all the bridge contracts have been deployed, send a native token from your Subnet to C-Chain with the [`cast send`](https://book.getfoundry.sh/reference/cast/cast-send) foundry command.
+
+```zsh
+cast send --rpc-url mysubnet --private-key $PK <tokenSourceAddress> "<functionToCall((parameterTypes))>" "(<functionParameter0>,<functionParameter1>,...)" --value <amountOfTokensToSend>
+```
+
+In line 60 of [`NativeTokenSource`](./NativeTokenSource.sol) is the send function we will call to send the tokens:
+
+```sol
+function send(SendTokensInput calldata input) external payable {
+        _send(input, msg.value, false);
+    }
+```
+
+The function parameters are defined by the `SendTokensInput` struct defined in line 26 of [`ITeleporterTokenBridge`](./interfaces/ITeleporterTokenBridge.sol).
+
+```sol
+struct SendTokensInput {
+    bytes32 destinationBlockchainID;
+    address destinationBridgeAddress;
+    address recipient;
+    address feeTokenAddress;
+    uint256 primaryFee;
+    uint256 secondaryFee;
+    uint256 requiredGasLimit;
```

Comment:
`feeTokenAddress` was assumed to be the address of the wrapped native token on Subnet
`requiredGasLimit` was arbitrarily set to 1000 

this command with these newly added parameters has not yet been tested.

Reply:
this has been tested